src/content/configuration/watch.md

```diff
@@ -158,4 +158,4 @@ On some machines Vim is preconfigured with the [backupcopy option](http://vimdoc
 
 ### Saving in WebStorm
 
-When using the JetBrains WebStorm IDE, you may find that saving changed files does not trigger the watcher as you might expect. Try disabling the `safe write` option in the settings, which determines whether files are saved to a temporary location first before the originals are overwritten: uncheck `File > Settings... > System Settings > Use "safe write" (save changes to a temporary file first)`.
+When using the JetBrains WebStorm IDE, you may find that saving changed files does not trigger the watcher as you might expect. Try disabling the `safe write` option in the settings, which determines whether files are saved to a temporary location first before the originals are overwritten: uncheck `File > Settings... > Appearance & Behavior > System Settings > Use "safe write" (save changes to a temporary file first)`.
```

Comment:
i have them in `WebStorm -> Preferences -> Appearance & Behavior -> ....` Maybe this is windows specific?

Reply:
"File > Settings..." for Windows, "File > Preferences..." for macOS on both [PhpStorm](https://www.jetbrains.com/help/phpstorm/system-settings.html) and [WebStorm](https://www.jetbrains.com/help/webstorm/system-settings.html)